lib/commands/release/start_release_command.dart

```diff
@@ -121,6 +121,25 @@ class StartReleaseCommand extends AlexCommand with IntlMixin {
       _commit("Generated translations.");
     }
 
+    final scriptPathes = config.preReliaseScripts?.scriptsPatches;
+
+    if (scriptPathes.isNotNullOrEmpty) {
+      printInfo('Running pre release cripts.');
+      for (final patch in scriptPathes!) {
+        final res = await flutter.runCmdOrFail(
+          'pub',
+          arguments: ['run', '${config.rootPath}/$patch'],
+        );
+        if (res.exitCode == 0) {
+          _commit('Run pre release cripts result.');
```

Comment:
1. 
Коммитить наверное логично в конце, после запуска всех скриптов? И тогда уже можно будет написать:
```suggestion
          _commit('Pre release scripts run.');
```

Там кстати он учитывает были изменения или нет? Не упадет если нечего коммитить?

2. После каждого выполнения лучше написать строчку в лог, типа `printInfo('Pre release script ${path} run - OK')`